src/compiler/sys.ts

```diff
@@ -1449,21 +1449,6 @@ declare const global: any;
 declare const __filename: string;
 declare const __dirname: string;
 
-export function getNodeMajorVersion(): number | undefined {

```

Comment:
I could leave this, but, it's always going to be in the git history.

Reply:
Wow, this was public. Why?

Reply:
Consider cherry-picking to release-5.0?

Reply:
This whole PR? Or should I just make the function internal so it looks like it's going away?